src/content/guides/creating-digital-tokens.mdx

```diff
@@ -32,22 +32,32 @@ sequenceDiagram
   Token Issuer ->> Centrapay: Create Collection
   Note left of Token Issuer: Confirm redeemable product<br/>IDs with merchant
   Token Issuer ->> Centrapay: Create Redemption Condition
-  Token Issuer ->> Centrapay: Create Token
-  Token Issuer ->> Centrapay: Transfer Token
+  alt
+    Note right of Token Issuer: Create token then send to user
+    Token Issuer ->> Centrapay: Create Token
+    Token Issuer ->> Centrapay: Transfer Token
+  else
+    Note right of Token Issuer: Or create token in user's account
+    Token Issuer ->> Centrapay: Create Token
+  end
 ```
 
 1. **Create a token collection**
 
    You can create Collections by calling our [Create Token Collection](/api/tokens#create-token-collection) endpoint. You will need the account identifier of the owning account for the `accountId` field. The `maxValue` field is optional, and defines the upper limit on any settlement amount for a redeemed token.
 
-2. **Create a redemption condition** for each merchant, using the collection Id from step 1.
+2. **Create a redemption condition** for each merchant, using the collection Id from step 1
 
    Redemption Conditions are created by calling the [Create Redemption Condition](/api/tokens#create-redemption-condition) endpoint. You will need the merchant identifier for the `merchantId` field. The list of `allowedProducts` defines those products sold by the merchant that are eligible for redemption. The `sku` for the product will need to match the `sku` that is passed in with the line item when the merchant creates the payment during redemption: [Create Payment Request](/api/payment-requests#create-a-payment-request).
 
-3. **Create tokens.**
+3. **Create tokens**
 
    Tokens can be created for the collection by calling the [Create Token](/api/tokens#create-token) endpoint.
 
-4. **Transfer the tokens to users.**
+4. **Transfer the tokens to users**
 
    You can send Tokens to users by calling our [Asset Transfers](/api/asset-transfers#create-an-asset-transfer) endpoint.
+
+5. **Create tokens directly into users' accounts**
```

Comment:
This shouldn't be the fifth step as you wouldn't do step 3 and 4 if you were doing this. I think instead we could have forking steps i.e. 3a and 3b.

Reply:
@Ray2126 Do you mean something like this? Mermaid doesn't allow manual numbering so unfortunately have to use the titles instead
![Screenshot 2023-11-20 at 1 47 21 PM](https://github.com/centrapay/centrapay-docs/assets/23620114/1740cc77-c49c-4f68-8fcd-1407a102716e)

Reply:
No, the diagram was fine. The numbering in this section that the comment is under is what I was referring to.